virttest/utils_disk.py

```diff
@@ -659,39 +693,74 @@ def configure_empty_windows_disk(session, did, size, start="0M",
 
     :param session: session object to guest.
     :param did: disk index which show in 'diskpart list disk'.
-    :param size: partition size. e.g. 500M
+    :param size: the total size of partitions. e.g. 2G
     :param start: partition beginning at start. e.g. 0M
     :param n_partitions: the number of partitions on disk
     :param fstype: filesystem type for the disk.
     :param labeltype: label type for the disk.
     :param timeout: Timeout for cmd execution in seconds.
     :return a list: mount point list for all partitions.
     """
+
+    def _get_availabel_size():
```

Comment:
a spelling mistake here, "availabel" should be "available", and if change to _get_disk_available_size() better?

Reply:
thanks, have updated.